zerver/lib/cache.py

```diff
@@ -8,6 +8,8 @@
 from django.conf import settings
 from django.db.models import Q
 
+from typing import Any, Iterable, Callable, Optional
```

Comment:
FWIW I've generally been sorting the `from typing` import lines, e.g. `from typing import Any, Callable, Iterable, Optional`.